client/my-sites/plans/current-plan/index.jsx

```diff
@@ -216,5 +229,6 @@ export default connect( ( state, { requestThankYou } ) => {
 		purchase: currentPlan ? getByPurchaseId( state, currentPlan.id ) : null,
 		showJetpackChecklist: isJetpackNotAtomic,
 		showThankYou: requestThankYou && isJetpackNotAtomic,
+		requestProduct: requestProduct,
```

Comment:
This can be simplified:

```suggestion
		requestProduct,
```